codecov.yml

```diff
@@ -3,7 +3,7 @@ coverage:
   status:
     patch:
       default:
-        target: 95
+        target: 75
     project:
       default:
-        target: 83
+        target: 75
```

Comment:
should not be less then 80 IMHO

Reply:
I agree that it should be as high as possible. Ideally.
The situation at present - in the last ticket we had coverage at 76.31%. If we set it higher (e.g. 80%) then all PRs will automatically fail (except those specifically targeted at increasing coverage).
My concern is that people will get into a habit of ignoring the code coverage test  - "it always fails, just ignore it".
It could be described as a "Normalization of deviance" situation, and the long term effect would be that _drops_ in coverage would be missed.
Re. your suggestion to set the figure at 80 - I don't think it will make any change to the current situation, and it negates the goal of this PR.